AppCenter/AppCenter/Internals/HttpClient/MSHttpClient.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSHttpClientProtocol.h"
+#import <Foundation/Foundation.h>
+
+@interface MSHttpClient : NSObject <MSHttpClientProtocol>
+
+/**
+ * Creates an instance of MSHttpClient.
+ *
+ * @return A new instance of MSHttpClient.
+ */
+- (instancetype)init;
+
+/**
+ * Creates an instance of MSHttpClient.
+ *
+ * @param maxHttpConnectionsPerHost The maximum number of connections that can be open for a single host at once.
```

Comment:
nit: "can be open" -> "can be opened"

Reply:
I think open is correct here

Reply:
https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=2ahUKEwivjczrjZThAhWMtZ4KHRYlDE8QFjABegQICxAF&url=https%3A%2F%2Fenglish.stackexchange.com%2Fquestions%2F141547%2Fopened-vs-open&usg=AOvVaw2fH-SCNB6HlepdwH-ygEE9

Reply:
That's not a verb. 😄